mlrun/utils/helpers.py

```diff
@@ -1015,16 +1015,33 @@ def create_class(pkg_class: str):
     return class_
 
 
-def create_function(pkg_func: str):
+def create_function(pkg_func: str, reload_modules: bool = False):
     """Create a function from a package.module.function string
 
     :param pkg_func:  full function location,
                       e.g. "sklearn.feature_selection.f_classif"
+    :param reload_modules: reload the function if it is in the modules list.
     """
     splits = pkg_func.split(".")
     pkg_module = ".".join(splits[:-1])
     cb_fname = splits[-1]
+    pkg_module_exists_in_modules = pkg_module in sys.modules
     pkg_module = __import__(pkg_module, fromlist=[cb_fname])
+
+    if reload_modules:
+        # if the function appears in the modules list, we need to reload the code again because it may have changed
+        if pkg_module_exists_in_modules:
```

Comment:
since we have opt-in `reload_modules` feature. I think we can remove this condition and execute it regardless to whether it apeear on modules or not.